plugins.json

```diff
@@ -88,6 +88,20 @@
 			"author": "Avery",
 			"minCrankshaftVersion": "0.2.1",
 			"source": "https://git.sr.ht/~avery/custom-library-tabs"
+		},
+		{
+			"id": "steamdeckhq-crankshaft-plugin",
+			"repo": "https://github.com/shelbykleindesign/steamdeckhq-crankshaft-plugin",
+
+			"version": "0.2.0",
```

Comment:
This should match the version in your `plugin.toml` (looks like it's 1.0 in the archive)